### PR TITLE
Mkdfs hometown file

### LINKED ISSUE #14: mkdfs checklist
- [x] Create an issue
- [x] Create branch
- [x] Create a PR
- [x] Collaborate!
- [x] Merge

### PR DESCRIPTION
**Pull request by mkdfs**
Adding hometown file for Chicago

Closes #14

### CODE REVIEW

File: mkdfs-chicago.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Chicago, IL
+
+## Places to Eat
+
+- Al's Beef 
+- Giordano's :pizza:
+
+- New Restaurant
```
Comment (akc020) [inline mkdfs-chicago.md]: which new restaurant?

**Review by akc020 — DISMISSED** [review]
Had some questions

File: mkdfs-chicago.md (hunk 1 above)
Comment (mkdfs) [reply mkdfs-chicago.md]: Don't know yet.  This is a placeholder 😸

**Comment by mkdfs** [comment]
Made changes to replace placeholder with "Portillo's"

**Review by beardofedu — APPROVED** [review]

**Pull request merged**